nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -497,8 +497,8 @@ sub in {
     $modes =~ m/([0-9]+)x([0-9]+)/;
     my $console_width = $1, my $console_height = $2;
     if ($console_width > 1920) {
-        push @attrs, "# High-DPI console";
-        push @attrs, 'console.font = lib.mkDefault "${pkgs.terminus_font}/share/consolefonts/ter-u28n.psf.gz";';
+        push @attrs, "# high-resolution display";
+        push @attrs, 'hardware.video.hidpi.enable = lib.mkDefault true;';
```

Comment:
I'm not sure having this is the best heuristic for HiDPI - what if the monitor is just really large? In general I don't think it's correct to determine HiDPI from resolution

Reply:
IMHO it's a reasonable assumption that if we have lots of pixels it's okay to make stuff bigger, even if it's not correct in absolutely all cases.

Reply:
If you have a better method. i.e. a simple way to decode edid, we can change it.
Notice that this PR does not change the way hidpi is detected.